src/NzbDrone.Core/MediaFiles/MediaInfo/MediaInfoFormatter.cs

```diff
@@ -71,6 +72,26 @@ public static string FormatAudioCodec(MediaInfoModel mediaInfo, string sceneName
 
             if (audioFormat.EqualsIgnoreCase("DTS"))
             {
+                if (audioAdditionalFeatures.StartsWithIgnoreCase("XLL"))
```

Comment:
I just found an example:

```
Scene Name:                          DTSHD-HRA
Format:                                   DTS
Format_Commercial_IfAny:    DTS-HD High Resolution Audio
Format_AdditionalFeatures:   ES XXCH XBR
CodecID:                                A_DTS
```

So it looks like additional features can contain multiple, but space separated

Reply:
I'd use a regex: `\bES XXCH\b` would match as expected so would `\bXLL X\b`, although the later one isn't apparent from the github links.

Reply:
doh! good point, I should have thought to look at the mediainfo source. Yeah it looks like the `X` part of `DTS XLL X` is the final `else` clause. If part of the profile doesn't match any of the others it just adds it to the additional features.

Reply:
Now it's my turn to say 'doh!' coz I didn't notice that the else would've done that. But that does mean that it's possible that it could potentially be `XLL ES X`, so XLL and X need to be matched separately.

Reply:
ok, i added the original code back in